gcloud/bigtable/test_client.py

```diff
@@ -16,6 +16,150 @@
 import unittest
 
 
+class Test__make_data_stub(unittest.TestCase):
+
+    def _callFUT(self, client):
+        from gcloud.bigtable.client import _make_data_stub
+        return _make_data_stub(client)
+
+    def test_it(self):
+        from gcloud._testing import _Monkey
+        from gcloud.bigtable import client as MUT
+
+        credentials = _Credentials()
+        user_agent = 'you-sir-age-int'
+        client = _Client(credentials, user_agent)
+
+        fake_stub = object()
+        make_stub_args = []
+
+        def mock_make_stub(*args):
+            make_stub_args.append(args)
+            return fake_stub
+
+        with _Monkey(MUT, make_stub=mock_make_stub):
+            result = self._callFUT(client)
+
+        self.assertTrue(result is fake_stub)
```

Comment:
Copy-pasta. We didn't always have `unittest2` (pre-Python 2.6 being dropped). Easy to change but you'd have to make a lot of comments like that to make it through all of our offenders in tests.